projects/lights_control/src/lights_config.c

```diff
@@ -0,0 +1,103 @@
+#include "misc.h"
+
+#include "gpio.h"
+#include "lights_config.h"
+#include "lights_events.h"
+
+#define CONVERT_TO_MASK(index) 1 << index
```

Comment:
Wrap your macro parameters and the entire thing please. Technically, the naming scheme doesn't match our style guide.